### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by asizemore**
Addresses #76 

Anything else we should add? Other wording tweaks? I figured the beeswarm can just live in a draft form until we need it.

### CODE REVIEW

**Comment by d-callan** [comment]
this looks really great :)

**Comment by d-callan** [comment]
reminder to merge #92 first, and update any new tests here to use the new dataDF

File: R/class-plotdata-beeswarm.R [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+#' Beeswarm Plot as data.table
+#'
+#' This function returns a data.table of 
+#' plot-ready data with one row per group (per panel). Columns 
+#' 'x', 'min', 'q1', 'median', 'q3' and 'max' represent the 
+#' pre-computed values per group. Columns 'group' and 'panel' specify
+#' the group the data belong to. 
+#' Optionally, can return columns 'outliers' and 'mean' as well.
+#' 
+#' @section Evil Mode:
```
Comment (d-callan) [inline R/class-plotdata-beeswarm.R:134]: think we should have the map section here too

File: README.md [hunk 2]
```diff
@@ -8,20 +8,131 @@ Use the R package [remotes](https://cran.r-project.org/web/packages/remotes/inde
...
+
+# or to install a specific version
+remotes::install_github('VEuPathDB/plot.data', 'v1.2.3')
 ```
+<br/>
 
 ## Usage
 
+All `plot.data` functions require at least the following arguments:
+1. A data frame or data table with columns corresponding to variables and rows to samples.
```
Comment (d-callan) [inline README.md:20]: idk if samples is the best word here. records? observations?

Comment (asizemore) [reply README.md:20]: Covered all bases -- changed it to "...columns corresponding to variables and rows to records (for example, observations, samples, etc.)"

**Comment by asizemore** [comment]
Updates:
- all plotting functions have map section
- all plotting functions (except heatmap) have examples
- beeswarm tests updated to use new organization and data. All tests passing.
- minor wording updates.

@d-callan ready for the last review!

**Review by d-callan — APPROVED** [review]

**Pull request merged**